index.js

```diff
@@ -1,11 +1,13 @@
 // ***** Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red")
+var counter = 1;
```

Comment:
`var` works here, but get in the habit of using `let` instead (`var` has some tricky behavior, so you won't see it used as much in modern Javascript).